### DISCUSSION TITLE
The "Pull Requests" tab on my repository settings is missing

### DISCUSSION THREAD

**Discussion by jasonfb** [community/community#65506] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

as per these instructions:
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

when I navigate to a _repository_ settings, I should see a settings choice (on the lefthand nav) for "Pull Requests"

However, strangely, I see no such choice. 

I am the owner of the repo, so I'm not sure if the docs are out of date or there is something else I'm doing wrong. 

Please advise. 

![Actions secrets 2023-08-30 17-26-06](https://github.com/community/community/assets/59002/7fd15b70-cd07-4dfb-a441-79171e9017d5)

**Comment by seyLu** [comment]
Hi @jasonfb 

#### I think what the docs meant, from the settings tab (by default it is on section `General`)
- scroll down
- and you would find the settings for pull requests

![image](https://github.com/community/community/assets/98249191/9c34ef80-1de1-41c6-a82b-91f5cc969066)
![image](https://github.com/community/community/assets/98249191/b19dd1c8-2e23-4f8d-a37f-3e3b84c06ed0)

#### But I do get that the github docs for that might need to be updated/improved somehow

**Comment by jasonfb** [comment]
thank you.

Documentation updated here https://github.com/github/docs/pull/27909
